lib/states.js

```diff
@@ -60,7 +60,7 @@ export default class States {
       {state: method}
     ).then(results => {
       if (errors.length > 0) {
-        throw new Error(errors.map(e => e.toString()).join(' | '))
+        throw new Error(errors.map(e => (e.toString() + e.stack)).join(' | '));
```

Comment:
But aren't the stack traces of these errors lost forever when we use e.toString()? Or am I misunderstanding something here?